sdk/sdk-view/src/test/java/mil/emp3/api/mock/MockMapInstance.java

```diff
@@ -204,6 +205,26 @@ public boolean validateCamera(double altitude, IGeoAltitudeMode.AltitudeMode alt
 
         return result;
     }
+
+    public boolean validateApplyCamera(double altitude, IGeoAltitudeMode.AltitudeMode altitudeMode, double latitude,
```

Comment:
since this returns a boolean the method name should be a question.

Reply:
This is too much bs.  The other validate methods are all boolean and I am keeping that consistent.